### PR TITLE
Some things I had issues with in refactor branch.

### PR DESCRIPTION
**Pull request by sandyhider**
- changed boss-config.py to be an executable, 
- added more to InitialAwsSetup.md 
- added to requirements.txt to import jsonschema to requirements
- needed by lib/configuration modules 
- removed chrony from lambda build sls

### CODE REVIEW

File: docs/InitialAwsSetup.md [hunk 1]
```diff
@@ -1,13 +1,122 @@
...
+to the bastion server. This is helpful when creating multiple bossdb stacks, example
+production, develop and testing stacks.  
+
+### Create a bastion key pair.
+* In the AWS Console under EC2 - Key Pairs: "Create Key Pair" for the bastion server
+
+### Create a new key pair for the first bosslet
+* In the AWS Console under EC2 - Key Pairs: "Create Key Pair" for the <your_bosslet_name>
+
+Move both key pairs to the ~/.aws directory. Change the name to end in .pem instead of .pem.txt if that is what it currently is.
```
Comment (derek-pryor) [inline docs/InitialAwsSetup.md]: `~/.ssh/` directory

File: docs/InitialAwsSetup.md [hunk 2]
```diff
@@ -1,13 +1,122 @@
...
+* In the AWS Console under EC2 - Key Pairs: "Create Key Pair" for the <your_bosslet_name>
+
+Move both key pairs to the ~/.aws directory. Change the name to end in .pem instead of .pem.txt if that is what it currently is.  
+Change the properties of both pem files to 400
+```sh
+chmod 400 <name>.pem
+``` 
+
+### Create new instance for Bastion
+To create a bastion server create a new t3.micro with thest latest release of ubuntu 14.04,
```
Comment (derek-pryor) [inline docs/InitialAwsSetup.md]: It doesn't have to be an Ubuntu based machine, they just need to set the appropriate username in the bosslet config.

File: docs/InitialAwsSetup.md [hunk 3]
```diff
@@ -1,13 +1,122 @@
...
+```sh
+$ git clone --recursive https://github.com/jhuapl-boss/boss-manage.git 
+```
+
+Create a virtual environment for working with bossdb and pip install the requirements in it.
+```sh
+$ pip install -r boss-manage.git/requirements.txt
+```
+
+### Create a boss-manage/config/lambda-build.json file with the following information filled out.
```
Comment (derek-pryor) [inline docs/InitialAwsSetup.md]: No longer needed

File: docs/InitialAwsSetup.md [hunk 4]
```diff
@@ -1,13 +1,122 @@
...
+    "aws_secret_key": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+    "aws_account": "123456789012",
+    "aws_bastion_ip": "xxx.xxx.xxx.xxx",
+    "aws_bastion_user": "ubuntu",
+    "aws_bastion_priv_key_file": "./bastion-key.pem"
+}
+```
+
+### build Lambda Build Server AMI using Packer
+```sh
```
Comment (derek-pryor) [inline docs/InitialAwsSetup.md]: `bin/packer.py <bosslet_config> lambda --ami-version "" -f`

File: requirements.txt [hunk 5]
```diff
@@ -7,3 +7,4 @@ pyminifier
 funcparserlib
 iso8601
 colorama
+jsonschema
```
Comment (derek-pryor) [inline requirements.txt]: What is using this?

Comment (sandyhider) [reply requirements.txt]: the path in my IDE must be pulling in the wrong things.  What was using jsonschema was /user-scratch/hiderrt1/scrape_boss/validator.py.  I need to track down why it was loading this validator.py in the first place.

Comment (sandyhider) [reply requirements.txt]: Removed the /user-scratch directories that were listed as sources in my IDE and it worked without jsonschema

**Pull request merged**